lib/EnsEMBL/REST/Controller/VEP.pm

```diff
@@ -304,6 +304,7 @@ sub _include_user_params {
     transcript_id
     phastCons
     distance
+    ambiguity
```

Comment:
FYI I've been doing some work on cleaning up this kind of code on a branch. Depending on who is finished first, there will be some nasty merge conflicts to attend to